SPECS/tdnf/tdnf-add-download-command.patch

```diff
@@ -297,6 +296,22 @@ index 871dae7..1bee9f5 100644
      pCmdArgs->nAllowErasing  = pCmdArgsIn->nAllowErasing;
      pCmdArgs->nAssumeNo      = pCmdArgsIn->nAssumeNo;
      pCmdArgs->nAssumeYes     = pCmdArgsIn->nAssumeYes;
+@@ -116,6 +117,15 @@ TDNFCloneCmdArgs(
+         BAIL_ON_TDNF_ERROR(dwError);
+     }
+ 
++    if(!IsNullOrEmptyString(pCmdArgsIn->pszDestDir))
++    {
++       dwError = TDNFAllocateString(
++                     pCmdArgsIn->pszDestDir,
++                     &pCmdArgs->pszDestDir);
++       pCmdArgs->nDownloadOnly=1;
```

Comment:
Is the presence of a destination directory sufficient to set downloadonly=1?
There is a "--downloadonly" command line argument (I believe) ..and there is a separate "download" command

Reply:
Destdir is only used in conjunction with the download command. Some of the download behavior depends on  nDownloadOnly being set to 1, so I decided to set it here. Destdir is a required argument for the download command.